compiler/env/VerboseLog.cpp

```diff
@@ -63,18 +63,19 @@ const char * TR_VerboseLog::_vlogTable[] =
    "#PROFILING: ",
    "#JITServer: ",
    "#AOTCOMPRESSION: ",
+   "FSD: ",
    };
 
 void TR_VerboseLog::writeLine(TR_VlogTag tag, const char *format, ...)
    {
-   TR_ASSERT(tag != TR_Vlog_null, "TR_Vlog_null is not a valid Vlog tag");
-   va_list args;
-   va_start(args, format);
-   writeTimeStamp();
-   write(_vlogTable[tag]);
-   vwrite(format,args);
+   write(tag, format);
```

Comment:
Don't you need to pass in the `va_list args` to `write`? Right now it would just be passing in the format string right?

Reply:
This is a mistake. Thanks for catching that. Fixed in [force-push](https://github.com/eclipse/omr/compare/9eb14ea976796dc8bc55ed6a33c06cbe0877cb71..e1723bfb4b4cd7f83ff7ff831396a4a3ba7a72cd).